### PR TITLE
Use database-backed minter from AF::Noid

### PR DESCRIPTION
**Pull request by mjgiarlo**

### CODE REVIEW

File: db/migrate/20170106175305_rename_minter_state_random_to_rand.active_fedora_noid_engine.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# This migration comes from active_fedora_noid_engine (originally 20161021203429)
+# frozen_string_literal: true
+class RenameMinterStateRandomToRand < ActiveRecord::Migration
+  def change
+    rename_column :minter_states, :random, :rand
```
Comment (jcoyne) [inline db/migrate/20170106175305_rename_minter_state_random_to_rand.active_fedora_noid_engine.rb]: Do you want to merge this migration with the other one?  That's few migrations to track.

File: config/initializers/active_fedora-noid.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+require 'active_fedora/noid'
+
+ActiveFedora::Noid.configure do |config|
+  config.minter_class = ActiveFedora::Noid::Minter::Db
```
Comment (jcoyne) [inline config/initializers/active_fedora-noid.rb]: Doesn't the Hyrax config take care of this for us?

Comment (mjgiarlo) [inline config/initializers/active_fedora-noid.rb]: Now it does, yeah.

File: db/migrate/20170106175305_rename_minter_state_random_to_rand.active_fedora_noid_engine.rb (hunk 1 above)
Comment (mjgiarlo) [inline db/migrate/20170106175305_rename_minter_state_random_to_rand.active_fedora_noid_engine.rb]: Sure.

**Comment by mjgiarlo** [comment]
@jcoyne ready for another review.

**Pull request merged**